docker/mariadb/00_database-create-geokrety-db.sql

```diff
@@ -526,42 +456,298 @@ CREATE TABLE `gk-waypointy` (
   `kraj` varchar(200) COLLATE utf8_polish_ci NOT NULL,
   `link` varchar(255) COLLATE utf8_polish_ci NOT NULL,
   `status` tinyint(4) NOT NULL DEFAULT '1',
-  `timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
-) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_polish_ci COMMENT='Waypoints';
+  `timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY (`waypoint`),
+  UNIQUE KEY `waypoint` (`waypoint`),
+  KEY `name` (`name`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_polish_ci;
 
--- --------------------------------------------------------
 
---
--- Table structure for table `gk-waypointy-gc`
---
+DROP TABLE IF EXISTS `gk-waypointy-country`;
+CREATE TABLE `gk-waypointy-country` (
+  `kraj` varchar(200) COLLATE utf8_polish_ci NOT NULL,
+  `country` varchar(200) COLLATE utf8_polish_ci DEFAULT NULL,
+  UNIQUE KEY `unique_kraj` (`kraj`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_polish_ci;
 
+INSERT INTO `gk-waypointy-country` (`kraj`, `country`) VALUES
+('Afghanistan',	'Afghanistan'),
+('Ägypten',	'Egypt'),
+('Albania',	'Albania'),
+('Albanien',	'Albania'),
+('Amerikanisch-Ozeanien',	'American Samoa'),
+('Argentinien',	'Argentina'),
+('Argentyna',	'Argentina'),
+('Armenia',	'Armenia'),
+('AT',	'Austria'),
+('AU',	'Australia'),
+('Australien',	'Australia'),
+('Austria',	'Australia'),
```

Comment:
For some reason previous comments were not added to the lines as expected... I meant here

Reply:
> Austria

Fixed in database.

> Sounds like extra useless key :)

No context, I don't understand sorry.

> why not remove Polish comment and keep English?

Yep something to be done.

> I don't know the typical queries against this table, but as per amount of indexes looks like introducing some composite indexes may be helpful

Sounds like you're talking about `gk-ruchy` table, no?

>  Is there any solid reason to have mixed English, German, Polish and abbreviation translations in one big heap in the table?

You are probably talking about `gk-waypointy-type` table. All those are strings imported from all imports from other caching platforms. That were not standardized on initial import. So this was a nice and easy workaround to standardize them and have them translated.